src/Microsoft.Azure.NotificationHubs/MpnsNotification.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.Azure.NotificationHubs
     /// </summary>
     public sealed class MpnsNotification : Notification, INativeNotification
     {
+        const string contentType = "application/xml";
```

Comment:
As noted in [our own documentation](https://docs.microsoft.com/en-us/rest/api/notificationhubs/send-mpns-native-notification) it should be `application/xml;charset=utf-8`